tests/foreman/ui/test_activationkey.py

```diff
@@ -269,6 +269,124 @@ def test_positive_create_with_host_collection(self):
                 (strategy, value % host_col.name))
             self.assertIsNotNone(host_collection)
 
+    @tier2
+    def test_positive_add_host_collection_non_admin(self):
+        """Test that hosts can be associated to Activation Keys by non-admin
+        user.
+
+        @id: 417f0b36-fd49-4414-87ab-6f72a09696f2
+
+        @Assert: Activation key is created, added host collection is listed
+
+        @CaseLevel: Integration
+        """
+        name = gen_string(str_type='alpha')
```

Comment:
Copy-paste from test above, thanks for pointing out.
>  it was never used in that way

It was but in 2014 according to blame :D